### PR TITLE
[Don't merge] Preview of changes in German rom

### PR DESCRIPTION
**Pull request by marijnvdwerf**

### CODE REVIEW

File: asm/crt0.s [hunk 1]
```diff
@@ -30,6 +30,41 @@ GPIOPortReadEnable: @ 80000C8
 
 	.space 6
 
+Unk1:
```
Comment (marijnvdwerf) [inline asm/crt0.s]: @YamaArashi Any idea what this might be?

Comment (YamaArashi) [inline asm/crt0.s]: I'm not sure. There's similar data in Pokemon Emerald. It seems to be mostly pointers to various things in the ROM.

File: asm/pokemon_3.s [hunk 2]
```diff
@@ -332,15 +331,15 @@ sub_803FB68: @ 803FB68
 	movs r1, 0x3
 	mov r2, sp
 	bl GetMonData
-	strb r0, [r4, 0x0]
+	strb r0, [r4]
 	lsls r0, 24
 	lsrs r0, 24
-	cmp r0, 0x2
+	cmp r0, 0x5
```
Comment (marijnvdwerf) [inline asm/pokemon_3.s]: `GAME_LANGUAGE`

File: asm/egg_hatch.s [hunk 3]
```diff
@@ -122,7 +122,7 @@ _0804297C:
 	lsrs r4, r0, 24
 	cmp r4, 0x5
 	bls _0804297C
-	movs r0, 0x2
+	movs r0, 0x5
```
Comment (marijnvdwerf) [inline asm/egg_hatch.s]: `GAME_LANGUAGE`

File: asm/pokemon_3.s [hunk 4]
```diff
@@ -2741,7 +2699,7 @@ sub_8040D3C: @ 8040D3C
 	cmp r1, 0x1D
 	bne _08040D7E
 _08040D54:
-	cmp r2, 0x2
+	cmp r2, 0x5
```
Comment (marijnvdwerf) [inline asm/pokemon_3.s]: GAME_LANGUAGE

File: asm/rom_810CBB4.s [hunk 5]
```diff
@@ -1564,7 +1564,7 @@ _0810D780:
 	movs r2, 0x1
 	movs r3, 0x2
 	bl ConvertIntToDecimalStringN
-	movs r0, 0xAD
+	movs r0, 0xB8
```
Comment (marijnvdwerf) [inline asm/rom_810CBB4.s]:
```
'.'         = AD
','         = B8
```

File: asm/rom_804373C.s [hunk 6]
```diff
@@ -1207,7 +1207,7 @@ _0804404C:
 	movs r0, 0x4
 	strb r0, [r3]
 	adds r3, 0x1
-	movs r0, 0xF0
+	movs r0, 0xAD
```
Comment (marijnvdwerf) [inline asm/rom_804373C.s]:
```
':'         = F0
'.'         = AD
```

File: asm/pokemon_3.s [hunk 7]
```diff
@@ -307,7 +307,6 @@ _0803FB56:
 	bx r0
 	thumb_func_end DrawSpindaSpots
 
-	.if REVISION >= 1
```
Comment (marijnvdwerf) [inline asm/pokemon_3.s]: @YamaArashi Except for the `GAME_LANGUAGE` candidate, is this function actually any different?

**Comment by marijnvdwerf** [comment]
Overview of functions that still need to be decompiled: 
asm/battle_message.s
- [ ] sub_8120AA8
- [ ] sub_8120FFC
- [ ] sub_8121A68

asm/battle_records.s
- [ ] ShowLinkBattleRecords

asm/berry_blender
- [ ] sub_804E884 (`GAME_LANGUAGE`)
- [ ] sub_8050E30
- [ ] sub_8051C58
- [ ] sub_805201C
- [ ] ShowBerryBlenderRecordWindow

asm/cable_club.s
- [ ] sub_808303C
- [ ] sub_8083314

asm/contest_link_7C
- [ ] sub_80C8604
- [ ] sub_80C86A0 (`GAME_LANGUAGE`)
- [ ] sub_80C8734
- [ ] sub_80C8938

asm/contest_link_80C02020.s
- [ ] de_sub_80C39A8
- [ ] de_sub_80C3A84
- [ ] sub_80C37E4

asm/decoration.s
- [ ] sub_80FECE0


asm/easy_chat
Differences all over file

asm/egg_hatch.s
- [ ] CreatedHatchedMon (`GAME_LANGUAGE`)

asm/item_menu.s
- [ ] sub_80A444C

asm/mon_markings.s
- [ ] sub_80F761C

asm/naming_screen.s
- [ ] sub_80B5F70
- [ ] sub_80B62CC
- [ ] sub_80B67EC
- [ ] sub_80B6878
- [ ] sub_80B6958
- [ ] sub_80B6998

asm/pokedex_cry_screen.s
- [ ] ShowPokedexCryScreen

asm/pokedex.s
- [ ] sub_8090750
- [ ] sub_8091304
- [ ] unref_sub_80913A4
- [ ] sub_8091458
- [ ] sub_8091564
- [ ] sub_8091E20

asm/pokemon_3.s
- [ ] sub_803FB68
- [ ] sub_8040D3C (`GAME_LANGUAGE`)

asm/pokemon_storage_system
- [ ] sub_8098898

asm/pokenav.s
- [ ] sub_80F700C

asm/rom_800D42C.s
- [ ] sub_803037C

asm/rom_810CBB4.s
- [ ] DetermineCyclingRoadResults

asm/rom_81258BC.s
- [ ] sub_812BB10
- [ ] sub_8133EF8
- [ ] sub_813545C
- [ ] de_sub_81364AC
- [ ] de_sub_81364F8

asm/rom_804373C.s
- [ ] sub_8043FC0

asm/secret_base.s
- [ ] sub_80BC190

asm/slot_machine.s
- [ ] sub_8104CAC
- [ ] sub_8106448

asm/trade.s
- [ ] sub_8049ED4
- [ ] sub_804A51C

**Pull request closed without merging**